plugins/ScheduledReports/templates/_addReport.twig

```diff
@@ -93,6 +93,13 @@
                  title="{{ 'ScheduledReports_EvolutionGraph'|translate(5)|e('html_attr') }}">
             </div>
         </div>
+        <div ng-show="manageScheduledReport.report.type == 'email'">
+            <div piwik-field uicontrol="checkbox"
+                 name="display_flat"
+                 ng-model="manageScheduledReport.report.displayFlat"
+                 title="{{ 'General_Flatten'|translate|e('html_attr') }}">
```

Comment:
I wonder if it is enough to show "Flatten" or whether could explain a little. Like Flatten will be only applied to reports that support it, and by default it is shown "hierarchical". Although I think in reports it is not really shown hierarchical by default but only the first level of the report which is not really useful so flatten is quite valuable. I think we need to explain it to users. @mattab  maybe an idea?